screens/Reader.js

```diff
@@ -485,8 +489,9 @@ function fadeInEffect() {
       dragging = true;
     });
     window.addEventListener('touchend', function() {
-      if(autoScrollSpeed != 0 && autoScrollTimeout == null) {
-       setAutoScroll();
+      if(autoScrollSpeed !== 0 && autoScrollTimeout === null) {
+        setTimeout(function() { window.ReactNativeWebView.postMessage('hide');},5000)
```

Comment:
Veerji, I previously I mean that this `setTimeout` can be refactored like this

```js
setTimeout(() => {
  window.ReactNativeWebView.postMessage('hide');
}, 5000)
```

But my suggestion is instead of indenting a piece of code in that file manually we should figure a way to turn on the formatter on this file. Because there are still some other indentation issues in this file.

Reply:
Veer ji this code is written in string. so we can not format the string. Either we have to rebuild this module or do it manually. 
You can view the code here veer ji. 
https://github.com/KhalisFoundation/sundar-gutka-react/blob/635501824e0d8086c76579827f31b9c5e2481888/screens/Reader.js